MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -387,6 +395,8 @@ public Person(String name, Campaign c, String factionCode) {
         bloodname = "";
         primaryDesignator = DESIG_NONE;
         secondaryDesignator = DESIG_NONE;
+        awardsFactory = AwardsFactory.getInstance();
```

Comment:
See note above about storing references to a singleton class in every instance of another class.